### ISSUE TITLE
[FEATURE] Download video in time range

### ISSUE THREAD

**Issue by TOMSLAUS** [Tzahi12345/YoutubeDL-Material#836]
Would be nice to have a feature where you can download not all of the video but just part of it by selecting video starting and ending time.

Edit: I apologize, I found this option within settings.

**Comment by Tzahi12345** [comment]
Yup the setting isn't super clear, ideally we could have a UI dedicated to it. How about we convert this issue to that feature request?

**Comment by TOMSLAUS** [comment]
Sounds good, in my experience lots of people use this feature. I think it should be converted to feature request.
